sql/catalyst/src/main/scala/org/apache/spark/sql/connector/expressions/expressions.scala

```diff
@@ -253,7 +253,7 @@ private[sql] final case class LiteralValue[T](value: T, dataType: DataType) exte
 }
 
 private[sql] final case class FieldReference(parts: Seq[String]) extends NamedReference {
-  import org.apache.spark.sql.connector.catalog.CatalogV2Implicits.MultipartIdentifierHelper
+  import org.apache.spark.sql.connector.catalog.CatalogV2Implicits.MultiPartIdentifierHelper
```

Comment:
This rename is not required and causes unnecessary code churn. Please revert it.

Reply:
This is obviously a typo and only affects this line, so I went ahead and fixed it. I can drop it if you think that's better, but it's probably that no one would send a PR to only fix this typo...

Reply:
I wouldn't say it's a typo. The word "multipart" without a space or a hyphen is by far [the most used spelling of multipart](https://books.google.com/ngrams/graph?content=multipart%2Cmulti-part%2Cmulti+part&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cmultipart%3B%2Cc0%3B.t1%3B%2Cmulti%20-%20part%3B%2Cc0%3B.t1%3B%2Cmulti%20part%3B%2Cc0#t1%3B%2Cmultipart%3B%2Cc0%3B.t1%3B%2Cmulti%20-%20part%3B%2Cc0%3B.t1%3B%2Cmulti%20part%3B%2Cc0).

I think this should be reverted.